tools/processresults.py

```diff
@@ -300,9 +303,12 @@ def _ReplaceUrlWithPlaceholder(results):
   """
   for r in results:
     q = r['responses'][0]['question'] # Do replacement in first question only
-    m = re.search('\"Proceed to.*?\"', q)
-    if m:
-      q = q.replace(m.group(0), '\"Proceed to [URL]\"')
-      r['responses'][0]['question'] = q
+    chosenMatch = re.search('\"Proceed to.*?\"', q)
```

Comment:
Is it possible for the response to include arbitrary characters? For example, could it be something like "Proceed to <script>alert(0)</script>" or something like that?

The client answers would be untrusted but I guess the server would reject malformed answers before reaching here, right?